contracts/network/contracts/network.sol

```diff
@@ -23,6 +23,8 @@ contract Network {
     // minimum deposit value to participate in the consensus
     uint public minDeposit = 100000;
 
+    bytes32 public votersSummary;
```

Comment:
I find the variable name confusing, had to check the implementation to find out it's more like a checksum. Maybe can be called `votersChecksum` or something more descriptive?.

Reply:
I agree, I had the same feeling about the name.